ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.ts

```diff
@@ -72,25 +78,33 @@ export class IndividualJourney extends JourneyBase {
     }
 
     let nextStep = this.stepOrder[currentStep + 1];
+    this.updateSubmitModelWithStep(currentStep);
 
     // access to a computer.
     if (this.model.computer === false) {
-      this.submit();
+      const modelToSave = this.stepsWithAnswers.find(m => m.step === currentStep);
+      modelToSave.model.computer = false;
```

Comment:
This looks strange to me, you're first checking if the `model.computer === false` and then you're setting it to false as well. If this is the case, is there a clearer way to sort that out?

Reply:
moved into a separate method into submit service